### PR TITLE
updated FAQ

### PR DESCRIPTION
**Pull request by lucianken**
adding new questions.

# [Title](https://clubhouseLink)

Updated FAQ per requested. 
  
## Changes 👷‍♀️

adding question specific to borrow. 
  
## How to test 🧪

Make sure I didn't break any code in the header.

### CODE REVIEW

**Review by FBrinkkemper — APPROVED** [review]
Approving just to check if that triggers heroku deployment

File: features/content/support/en.ts [hunk 1]
```diff
@@ -3,11 +3,93 @@ import { ContentTypeSupport } from './support'
 export const content: ContentTypeSupport = {
   title: 'FAQ',
   navigation: [
+    { title: 'Using Oasis Borrow', id: 'using-borrow' },
```
Comment (FBrinkkemper) [inline features/content/support/en.ts:6]: We now refer to Oasis Borrow as `Oasis`, and Oasis.app as `Daiwallet`. Please update the PR by removing Borrow from the app names and removing /borrow from the links

**Review by FBrinkkemper — CHANGES_REQUESTED** [review]
few changes to be made before merging

File: features/content/support/en.ts (hunk 1 above)
Comment (lucianken) [reply features/content/support/en.ts]: fixed

File: features/content/support/en.ts [hunk 2]
```diff
@@ -3,26 +3,108 @@ import { ContentTypeSupport } from './support'
...
+      title: 'Using Daiwallet',
+      id: 'using-daiwallet',
+      questions: [
+        {
+          question: 'What is Dai Wallet?',
+          answer: `Dai Wallet is the home for everything you want to accomplish with Dai. A decentralized application that runs on the Ethereum blockchain, Oasis Dai Wallet enables you to Buy, Send, and Manage your Dai all in one place.`,
         },
         {
           question: 'What is Dai?',
           answer: `Dai is a better, smarter digital currency for everyone. It is the world’s first unbiased currency and its value consistently tracks the US Dollar, which means it doesn't suffer from the volatility associated with many other digital currencies. To learn more about Dai, read our [short primer](/dai).`,
```
Comment (FBrinkkemper) [inline features/content/support/en.ts:103]: /dai -> /daiwallet/dai

File: features/content/support/en.ts [hunk 3]
```diff
@@ -78,7 +160,7 @@ export const content: ContentTypeSupport = {
           answer: `Oasis.app doesn't take any of the fees when you buy Dai or ETH through one of our partner providers. The fee you pay goes solely and directly to the third-party provider.`,
```
Comment (FBrinkkemper) [inline features/content/support/en.ts:160]: Oasis.app ... -> Our Dai Wallet ...

File: features/content/support/en.ts [hunk 4]
```diff
@@ -3,26 +3,108 @@ import { ContentTypeSupport } from './support'
...
+
+        {
+            question: 'How can I contact the Oasis team?',
+            answer:
+              'If you have any questions, reach out to us through our [Contact page](/contact) or on [Twitter](https://twitter.com/oasisdotapp).',
+          },
+
+
+    {
+      title: 'Using Daiwallet',
```
Comment (FBrinkkemper) [inline features/content/support/en.ts:94]: Using Daiwallet > Using Dai Wallet

File: features/content/support/en.ts [hunk 5]
```diff
@@ -3,26 +3,108 @@ import { ContentTypeSupport } from './support'
...
+
+        {
+          question: 'Why would I change the transaction speed?',
+          answer: `Transaction speed allows you to pay more gas to get your transactions mined faster. In case you are in a hurry, for example to increase your Collateralization Ratio to avoid liquidation, you can set a fast speed for your transactions.`,
+        },
+
+        {
+            question: 'How can I contact the Oasis team?',
+            answer:
+              'If you have any questions, reach out to us through our [Contact page](/contact) or on [Twitter](https://twitter.com/oasisdotapp).',
```
Comment (FBrinkkemper) [inline features/content/support/en.ts:89]: /contact -> /daiwallet/contact

File: features/content/support/en.ts [hunk 6]
```diff
@@ -3,26 +3,108 @@ import { ContentTypeSupport } from './support'
...
+        },
+
+        {
+          question: 'How much does it cost?',
+          answer: `Opening and managing a Vault is free on Oasis.app except for gas costs and Stability Fees. The Stability Fee is charged on the amount of Dai you have generated and goes directly to the Maker Protocol.`,
+        },
+
+        {
+          question: 'How do I open a Vault?',
+          answer: `To open a Vault, select the relevant Collateral and sub-type (e.g. ETH-A) from the homepage (Oasis.app/borrow) and connect your preferred wallet and follow the on screen instructions that will guide you through.`,
```
Comment (FBrinkkemper) [inline features/content/support/en.ts:28]: Oasis.app/borrow -> Oasis.app

**Comment by FBrinkkemper** [comment]
Created new PR within oasis-borrow repo and did some changes to make it work - https://github.com/OasisDEX/oasis-borrow/pull/315

**Pull request closed without merging**